type_templates/glb.js

```diff
@@ -561,16 +574,17 @@ export default e => {
         const localPlayer = useLocalPlayer();
 
         let sitAction = {
-            type: 'sit',
-            time: 0,
-            animation: rideSpec.subtype,
-            controllingId: instanceId,
-          };
+          type: 'sit',

```

Comment:
Btw why do we wait to the next frame to finalize the sit? That's what it seems like we're doing here.